docs/api/core.md

```diff
@@ -850,38 +850,21 @@ The request's data for `DeployService` API.
 **Example**
 ```json
 {
-  "service": {
-    "name": "serviceX",
-    "events": {
-      "eventX": {
-        "data": {
-          "dataX": { "type": "String" }
-        }
-      }
-    },
-    "tasks": {
-      "taskX": {
-        "inputs": {
-          "foo": { "type": "String" }
-        },
-        "outputs": {
-          "outputX": {
-            "data": {
-              "resX": { "type": "String" }
-            }
-          }
-        }
-      }
-    }
-  }
+  "url": "__SERVICE_GIT_URL__"
+}
+```
+or
+```json
+{
+  "url": "__SERVICE_GZIPPED_TAR_FILE_CHUNK__"
```

Comment:
this should be chunk no ?

Reply:
yes, I'm still working in this PR. I'll request review after completed.